__tests__/app.test.js

```diff
@@ -42,7 +42,47 @@ describe('GET /api', () => {
                 const arr = Object.values(body);
                 arr.forEach((item) => {
                     expect(item).toMatchObject(expectedShape);
-                })
-            })
+                });
+            });
+    });
+});
+
+describe('GET /api/articles/:article_id', () => {
+    test('200: responds with an article object', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .expect("Content-Type", /json/)
+        .then(({ body }) => {
+          const expectedShape = {
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: 1,
+            body: expect.any(String),
+            topic: expect.any(String),     
+            created_at: expect.any(String),          
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
```

Comment:
A bit pedantic, but if we are getting one specific article, we know what the title, topic, author etc. are looking like and can be more specific about what we want to get back